weeks/week02/HW2_Task3.py

```diff
@@ -0,0 +1,35 @@
+# List of steps:
+steps_1 = ["open browser", "navigate to page", "click login"]
+steps_2 = ["open browser", "navigate to page", "fill form"]
+steps_3 = ["open browser", "navigate to page", "click login"]  # duplicate steps
+
+# Use frozenset() to store the test steps of each scenario in an immutable set:
+scenario_1 = frozenset(steps_1)
+scenario_2 = frozenset(steps_2)
+scenario_3 = frozenset(steps_3)
+
+# Store each test scenario in a dictionary where the key is the test case name and the value is the frozenset of steps:
+dic_scenarios = {"Test Case 1": scenario_1, "Test Case 2": scenario_2, "Test Case 3": scenario_3, }
+print(type(dic_scenarios))
```

Comment:
Do you really want to have it here? I think you added it for debugging purposes, didn't you?